packages/ui/src/components/va-toast/useToast.ts

```diff
@@ -0,0 +1,37 @@
+import { createToastInstance, closeById, closeAllNotifications, NotificationOptions } from './toast'
+import { getCurrentInstance } from 'vue'
+
+/** This hook can be used without plugin used */
+export const useToast = () => {
+  const appContext = getCurrentInstance()?.appContext
+
+  const createdInThisSetupContext: string[] = []
+
+  /**
+   * @param options can be options object or message string
+   */
+  const init = (options: NotificationOptions | string) => {
+    const id = createToastInstance(options, appContext)
+
+    if (id) { createdInThisSetupContext.push(id) }
+
+    return id
+  }
+
+  const close = (id: string) => closeById(id)
+
+  /**
+   * Closes all VaToast instances in current App instance.
+   * @param allApps if you have multiple vue apps on page, set allApps to true to close it for all apps.
+   */
+  const closeAll = (allApps = false) => closeAllNotifications(allApps ? undefined : appContext)
+
+  /** Use this method if you need to close only toasts that created with this hook */
+  const closeAllCreatedInThisHook = () => {
+    createdInThisSetupContext.forEach((id) => closeById(id))
```

Comment:
```suggestion
    createdInThisSetupContext.forEach(closeById)
```

Reply:
Actually, passing id argument is much readable. Also, if we will add some additional argument to closeById method, this forEach can cause some bugs. I would prefer to use a new arrow function as callback.